tds/src/test/java/thredds/server/ncss/dataservice/FeatureDatasetTypeTest.java

```diff
@@ -110,4 +110,13 @@ public void getFDForFeatureCollectionStationDataset() throws IOException{
 		assertNotNull(fd);
 		assertEquals(FeatureType.STATION, fd.getFeatureType());
 	}
+	@Test
+	public void getFDForSimpleGeometryFeatureDataset() throws IOException{
+		MockHttpServletRequest req = new MockHttpServletRequest();
+		MockHttpServletResponse res = new MockHttpServletResponse();
+
+		FeatureDataset fd = TdsRequestedDataset.getSimpleGeometryFeatureDataset(req, res, "/thredds/cdm/src/test/data/dataset/SimpleGeos/hru_soil_moist_3hru_5timestep.nc");
```

Comment:
Some of the other tests in this class use short relative paths to access the data. Should this test follow suit?